assets/scss/checkout.scss

```diff
@@ -1,103 +1,3 @@
 // =============================================================================
 // Stencil Checkout  - Customize the checkout experience
 // =============================================================================
```

Comment:
If this file is only going to be used for Blueprint Checkout, maybe we could specify that?